frontend/src/plugins/router.js

```diff
@@ -69,6 +71,7 @@ export default new Router({
                 {
                     path: 'websites/add',
                     component: AddWebsitePage,
+                    redirect: { name: 'add_websites_step_1' },
```

Comment:
no quite sure if this works, if you need redirect to from default route to some internal route, use child with empty path (default path)